docs/antora.yml

```diff
@@ -0,0 +1,5 @@
+name: agent
+version: master
```

Comment:
The version could be set to `0.3.0-nightly` and be updated manually on every release.

Reply:
Yeah, that'd be better I agree.
I'd like to get the whole infrastructure set up first before thinking about versioning documentation.
At the moment no one's going to need old documentation I assume.